src/impl.cpp

```diff
@@ -716,16 +718,31 @@ std::ostream& operator<<(std::ostream& stream, const Manifold::Impl& impl) {
  * the tolerance and epsilon. Useful for debugging and testing.
  * Should be used with ImportMeshGL64 for reproducing issues.
  */
-std::ostream& Manifold::Dump(std::ostream& stream) const {
+std::ostream& Manifold::WriteOBJ(std::ostream& stream) const {
   return stream << *GetCsgLeafNode().GetImpl();
 }
 
+bool Manifold::WriteOBJ(std::string& filename) const {
+  if (!filename.length()) return false;
+  std::ofstream ofile;
+  ofile.open(filename);
+  if (!ofile.is_open()) return false;
+  WriteOBJ(ofile);
+  ofile.close();
+  return true;
+}
+
+bool Manifold::WriteOBJ(const char* filename) const {
```

Comment:
Is this necessary? I thought a std::string would accept a literal anyway.

Reply:
Hmm - I'd have to check a raw const char * to a std::string method would work, but it doesn't matter - I can just make the std::string on my end.  I'll yank 'em.